templates/js-crawlee-cheerio/README.md

```diff
@@ -1,12 +1,19 @@
-# CheerioCrawler Actor template
+# JavaScript Cheerio Actor template
 
-This template is a production ready boilerplate for developing with `CheerioCrawler`. Use this to bootstrap your projects using the most up-to-date code.
+A template example built with [Crawlee](https://crawlee.dev) to scrape data from a website using [Cheerio](https://cheerio.js.org/).
```

Comment:
I think this should mention the `CheerioCrawler` too https://crawlee.dev/api/cheerio-crawler/class/CheerioCrawler